libsolidity/interface/Natspec.h

```diff
@@ -40,6 +40,8 @@ struct DocTag;
 class Natspec
 {
 public:
+	static unsigned int constexpr NATSPEC_VERSION = 1;
```

Comment:
This might need a replacement. Maybe `c_natspecVersion`?

Reply:
@Marenz Sorry. It should be `s_` for static. https://github.com/ethereum/solidity/blob/develop/CODING_STYLE.md#4-variable-prefixes

Reply:
I've also seen `libevmasm/CommonSubexpressionEliminator.h:	static int const c_invalidPosition = -0x7fffffff` so there is at least a precendent for this